contracts/Nexus.sol

```diff
@@ -226,7 +227,9 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
     /// @dev Delegates the validation to a validator module specified within the signature data.
     function isValidSignature(bytes32 hash, bytes calldata signature) external view virtual override returns (bytes4) {
         // Handle potential ERC7739 support detection request
-        if (checkERC7739Support(hash, signature)) return SUPPORTS_ERC7739;
+        if (signature.length == 0) {
```

Comment:
if signature length is 0 then check? is this correct?

Reply:
yes, it's in 7739 spec